src/resources/websitetoolbox.js

```diff
@@ -0,0 +1,130 @@
+wtbx.setting = {
+
+    checkAll: function(e){
```

Comment:
Function name not correct. It looks like check all check-box . So rename the function name.